### PR TITLE
[5085] Clicking through to a plugin/widget doesn't work in Expreience Builder when pressing z

### PR DESCRIPTION
**Pull request by jvega190**
https://github.com/craftercms/craftercms/issues/5085

### CODE REVIEW

File: delivery/templates/youtube.ftl [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <#import "/templates/system/common/crafter.ftl" as crafter />
+<script src = "http://www.youtube.com/player_api"></script>
```
Comment (rart) [inline delivery/templates/youtube.ftl]: Should only be imported once despite the plugging being used multiple times on a page

**Review by rart — CHANGES_REQUESTED** [review]

File: delivery/templates/youtube.ftl (hunk 1 above)
Comment (jvega190) [reply delivery/templates/youtube.ftl]: I made some enhancements for a better handling of multiple videos in the same page.

**Review by rart — APPROVED** [review]

File: delivery/templates/youtube.ftl [hunk 2]
```diff
@@ -41,11 +43,30 @@
 
     <@crafter.div $field=contentModel
```
Comment (rart) [inline delivery/templates/youtube.ftl:44]: `$field=contentModel`? That's not right.

**Review by rart — CHANGES_REQUESTED** [review]

**Comment by rart** [comment]
Please close in favour of https://github.com/craftercms/youtube-plugin/pull/2

**Pull request closed without merging**